add clarity to docs (#238)
BrightspaceUI/testing: docs/migration-guide.md

### Step 4: Migrating CI Goldens

Finally! The last step is to automatically migrate the repo's CI goldens. To do this, run:

```bash
d2l-test-runner vdiff migrate ./**/<location>/*
```

Again, the optional location parameter can be used to migrate a subset of tests.

> Commit the changes to the goldens, which will be limited to files being moved/renamed and the addition of `.vdiff` to the repo's `.gitignore`.

Reviewer: It isn't really a change to the goldens so much as a new location.

```suggestion
Commit the moved goldens to the repo. Also add `.vdiff` to the repo's `.gitignore` file and commit.
```

Author: Cool thanks, went with a slight variation on this.
